strimzi/04-service-binding.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: kafka-producer-binding-user
+spec:
+  application:
+    apiVersion: app.stacks/v1beta1
+    kind: RuntimeComponent
+    name: kafka-producer
+  service:
+    apiVersion: v1beta1/kafka.strimzi.io
+    kind: KafkaUser
+    name: my-producer-user
+---
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: kafka-producer-binding-cluster
+spec:
+  application:
+    apiVersion: app.stacks/v1beta1
+    kind: RuntimeComponent
+    name: kafka-producer
+  service:
+    apiVersion: v1beta1/kafka.strimzi.io
+    kind: Kafka
+    name: my-producer-cluster
+    mappings:
+      - name: hosts
+        value: |-
+          {{- range .status.listeners -}}
```

Comment:
so happy we didn't define a custom grammar

Reply:
definitely!